app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsUtils.kt

```diff
@@ -17,21 +17,28 @@
 package com.example.android.architecture.blueprints.todoapp.statistics
 
 import com.example.android.architecture.blueprints.todoapp.data.Task
+import javax.inject.Inject
 
 /**
- * Function that does some trivial computation. Used to showcase unit tests.
+ * Class used to showcase dependency injection scoped to one module ([StatisticsModule]).
  */
-internal fun getActiveAndCompletedStats(tasks: List<Task>?): StatsResult {
+class StatisticsUtils @Inject constructor() {
```

Comment:
This doesn't need the `@Inject constructor()`

Reply:
I think it does. Unless I provide it in the module. What's the usual pattern here?

Reply:
Ah ok! Not sure about the usual pattern, but I'd provide it in the module only if it's scoped (so there's more context around it). Seem it's not scoped so it's ok to provide it in this way

Reply:
done